library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -3373,7 +3373,9 @@
     <string name="device_manager_learn_more_sessions_unverified_title">Unverified sessions</string>
     <string name="device_manager_learn_more_sessions_unverified">Unverified sessions are sessions that have logged in with your credentials but not been cross-verified.\n\nYou should make especially certain that you recognise these sessions as they could represent an unauthorised use of your account.</string>
     <string name="device_manager_learn_more_sessions_verified_title">Verified sessions</string>
+    <!-- TODO TO BE REMOVED -->
     <string name="device_manager_learn_more_sessions_verified">Verified sessions have logged in with your credentials and then been verified, either using your secure passphrase or by cross-verifying.\n\nThis means they hold encryption keys for your previous messages, and confirm to other users you are communicating with that these sessions are really you.</string>
+    <string name="device_manager_learn_more_sessions_verified_description">Verified sessions are anywhere you are using ${app_name} after entering your passphrase or confirming your identity with another verified session.\n\nThis means that you have all the keys needed to unlock your encrypted messages and confirm to other users that you trust this session.</string>
```

Comment:
Maybe we could use "anywhere you are using this account"? @jakewb-b WDYT?

Reply:
> anywhere you are using this account

Yes, that's ideal. Thanks for spotting that.

Reply:
Thanks, done.

Reply:
I think we should also warn other platforms of this change so that we use the same text everywhere to be consistent.

Reply:
@onurays in case you have done so already, could you update our internal task that tracks this to reflect the wording change? If some platforms have already done this task, they may need a separate ticket.